rust/noosphere-storage/src/lib.rs

```diff
@@ -25,11 +26,12 @@ pub use encoding::*;
 pub use implementation::*;
 pub use key_value::*;
 pub use retry::*;
+pub use scratch::*;
 pub use storage::*;
 pub use store::*;
 pub use tap::*;
 
-#[cfg(test)]
+#[cfg(feature = "helpers")]
 pub mod helpers;
```

Comment:
I don't think it's possible to export test-only exports between crates -- promoted to "helpers" here

Reply:
SGTM; I started doing this in `subconscious-cloud` the other day myself